source/core.py

```diff
@@ -178,19 +177,27 @@ def main():
 	log.debug("loading config")
 	import config
 	config.initialize()
-	if not globalVars.appArgs.minimal and config.conf["general"]["playStartAndExitSounds"]:
-		try:
-			nvwave.playWaveFile("waves\\start.wav")
-		except:
-			pass
-	logHandler.setLogLevelFromConfig()
 	try:
 		lang = config.conf["general"]["language"]
 		import languageHandler
 		log.debug("setting language to %s"%lang)
 		languageHandler.setLanguage(lang)
 	except:
 		log.warning("Could not set language to %s"%lang)
+	import NVDAHelper
+	log.debug("Initializing NVDAHelper")
+	NVDAHelper.initialize()
+	if not globalVars.appArgs.minimal:

```

Comment:
Is there a particular reason why you only unmute if not minimal?

Reply:
15.1. Command Line Options
-m
--minimal
No sounds, no interface, no start message, etc.
I'm thinking of also putting this behind a config value though, like you said. I need it to be available for shared environments though, because this might help people in a lab.